cmd/create/oidcconfig/cmd.go

```diff
@@ -94,10 +110,15 @@ func run(cmd *cobra.Command, argv []string) {
 	args.region = region
 
 	// Determine if interactive mode is needed
-	if !interactive.Enabled() && !cmd.Flags().Changed("mode") {
+	if !interactive.Enabled() && !cmd.Flags().Changed("mode") && !cmd.Flags().Changed(rawFilesFlag) {
 		interactive.Enable()
 	}
 
+	if args.rawFiles && mode != "" {
+		r.Reporter.Warnf("--raw-files param is not supported alongside --mode param.")
+		os.Exit(1)
+	}
+
 	if interactive.Enabled() {
```

Comment:
Should we enter here only if `args.rawFiles == false`?

Reply:
I see what you mean, if the user specifies -i he would get there alongside raw-files.